### PR TITLE
Support class static blocks

### PR DESCRIPTION
**Pull request by Shraymonks**
Add support for static blocks in classes. Fixes https://github.com/tree-sitter/tree-sitter-javascript/issues/222

References:
[MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Static_initialization_blocks)
[TC39 spec](https://tc39.es/ecma262/multipage/ecmascript-language-functions-and-classes.html#prod-ClassStaticBlock)

Checklist:
- [x] All tests pass in CI.
- [x] The script/parse-examples passes without issues.
- [x] There are sufficient tests for the new fix/feature.
- [x] Grammar rules have not been renamed unless absolutely necessary.
- [x] The conflicts section hasn't grown too much.
- [x] The parser size hasn't grown too much (check the value of STATE_COUNT in src/parser.c).

### CODE REVIEW

**Comment by aryx** [comment]
you can probably rebase on latest to fix the windows CI failure.

**Comment by Shraymonks** [comment]
> you can probably rebase on latest to fix the windows CI failure.

Rebased!

File: grammar.js [hunk 1]
```diff
@@ -1105,6 +1106,13 @@ module.exports = grammar({
       ')'
     ),
 
+    class_static_block: $ => seq(
+      'static',
+      '{',
+      repeat($.statement),
+      '}',
```
Comment (maxbrunsfeld) [inline grammar.js:1113]: Can you reuse the `statement_block` rule here?

**Review by maxbrunsfeld — COMMENTED** [review]
Thanks! Just one suggestion.

File: grammar.js (hunk 1 above)
Comment (Shraymonks) [reply grammar.js]: Done

**Pull request merged**